### PR TITLE
Add default image + some UI updates

### PR DESCRIPTION
**Pull request by keanecjy**

### CODE REVIEW

**Comment by keanecjy** [comment]
image

Tapping on the trip does not end up in the right trip somehow, might already be a problem that existed before.

File: TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift [hunk 1]
```diff
@@ -38,15 +38,7 @@ struct TripView: View {
                             .padding([.bottom, .horizontal])
 
                         if let imageUrl = viewModel.trip.imageUrl {
```
Comment (tanruiquan) [inline TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift:40]: `imageUrl` would always exist given our current implementation?

**Review by tanruiquan — APPROVED** [review]
LGTM 😄

File: TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift (hunk 1 above)
Comment (keanecjy) [reply TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift:40]: yep, but its an optional field now.. maybe we can change it in next sprint

**Pull request merged**